griptape/observability/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .observability import Observability
+
+__all__ = ["Observability"]
```

Comment:
Wondering if this should live in `griptape.utils` instead of introducing a new module. What do you think?

Reply:
We totally could, especially considering the amount of code in here is trivial. I originally added a new top level module because it felt like a top level feature (like memory) that could use the attention. None of the observability drivers are intended to be used without this module, and so it really is the primary entry point for the feature. That said, I don't really have much of a strong opinion.

Are you still on the fence?

Reply:
Hmm that's a good point. I'd like to be conscious of having one-off modules, but I also don't want to bloat `griptape.utils` What do you think @vasinov?

Reply:
I agree with Dylan's reasoning here. It feels like it should be its own thing, particularly because it has its own drivers.